src/pages/myGeotab/whatsNew.tsx

```diff
@@ -1,8 +1,3045 @@
+			</li>
+			<li>
+				<p><a href="/sdk/software/api/reference/#API">API.cs</a> (.Net only): Fix bug, in certian senario changing <code className="small-code-sample">Timeout</code> property could abort the action on timeout and not cancel underlying request.</p>
+			</li>
+			<li>
+				<p><a href="/sdk/software/api/reference/#CustomVehicleDevice">CustomVehicleDevice</a>: Support of vehicle specific custom devices which provide vehicle specific properties and functionality. Custom device product ID must be of CustomVehicleDevice type. Contact your reseller for more information.</p>
+				<ul>
+					<li>
+						<p>Improved support for calculated odometer and raw odometer with third-party diagnostic KnownId <code className="small-code-sample">DiagnosticThirdPartyOdometerId</code> and <code className="small-code-sample">OdometerAdjusmentId</code></p>
+					</li>
+					<li>
+						<p>Improved support for calculated engine hours with third-party diagnostic KnownId <code className="small-code-sample">DiagnosticThirdPartyEngineRunTimeId</code> and <code className="small-code-sample">EngineHoursAdjusmentId</code></p>
+					</li>
+					<li>
+						<p><code className="small-code-sample">VehicleIdentificationNumber</code> property moved from CustomDevice to CustomVehicleDevice</p>
+					</li>
+					<li>
+						<p>Added <code className="small-code-sample">LicencePlate</code> and <code className="small-code-sample">LicenceState</code> properties</p>
+					</li>
+				</ul>
+			</li>
+			<li>
+				<p><a href="/sdk/software/api/reference/#DutyStatusAvailability">DutyStatusAvailability</a>: Added BETA support for <code className="small-code-sample">Recap</code> and <code className="small-code-sample">CycleAvailabilities</code> properties</p>
+			</li>
+			<li>
+				<p><a href="/sdk/software/api/reference/#DutyStatusAvailability">DutyStatusAvailability</a>: Replaced <code className="small-code-sample">Availabilities</code> list with separate properties: <code className="small-code-sample">Driving</code>, <code className="small-code-sample">Cycle</code>, <code className="small-code-sample">Rest</code>, <code className="small-code-sample">Duty</code>, <code className="small-code-sample">Workday</code></p>
+			</li>
+			<li>
+				<p><a href="/sdk/software/api/reference/#DeviceType">DeviceType</a>: Added <code className="small-code-sample">CustomVehicleDevice</code>.</p>
+			</li>
+			<li>
+				<p><a href="/sdk/software/api/reference/#DriverChange">DriverChange</a>: DriverChange object Id property is no longer backed by integer type. It is now backed by GUID type. When update 1804 is applied to the database, all previous numeric entity Id’s will be invalidated and assigned a new GUID Id’s. This could pose an issue if your integration stores driver change Id and you then reference the DriverChange by that Id. Note: JSON representation of Id was previously string and remains string type.</p>
```

Comment:
The character U+2019 "’" could be confused with the ASCII character U+0060 "`", which is more common in source code

multiple instances, but just needs to be changed to '